ui/raidboss/data/05-shb/raid/e6s.js

```diff
@@ -494,6 +493,7 @@
     },
     {
       'locale': 'ja',
+      'missingTranslations': true,
```

Comment:
?

Reply:
I understand that this line `'missingTranslations'`: true, should be there, when there are lines to translate. Am I mistaken?
![e6s ja](https://user-images.githubusercontent.com/61565834/84476812-fab94200-ac8e-11ea-8c1c-3d44feea4557.PNG)